python/instrumentation/openinference-instrumentation-pydanticai/pyproject.toml

```diff
@@ -0,0 +1,105 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "openinference-instrumentation-pydanticai"
```

Comment:
I think if this is not actually an instrumentor, it might be worth naming this opeininference-pydanticai rather than instrumentation since it's not an instrumentor

Reply:
though if this just makes the symlinking confusing I think it's okay. Defer to @axiomofjoy